builtin/providers/aws/resource_aws_elasticache_replication_group.go

```diff
@@ -215,6 +220,8 @@ func resourceAwsElasticacheReplicationGroupRead(d *schema.ResourceData, meta int
 		d.Set("snapshot_window", c.SnapshotWindow)
 		d.Set("snapshot_retention_limit", c.SnapshotRetentionLimit)
 
+		d.Set("primary_endpoint_address", rgp.NodeGroups[0].PrimaryEndpoint.Address)
```

Comment:
Is there any possibility of this being empty?

Reply:
@stack72 @jen20  -How to get addresses for replica nodes?